api/v1/jobs/PKPJobHandler.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * @file api/v1/jobs/PKPJobHandler.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class DoiHandler
+ * @ingroup api_v1_jobs
+ *
+ * @brief Handle API requests for jobs
+ *
+ */
+
+namespace PKP\API\v1\jobs;
+
+use APP\facades\Repo;
+use PKP\core\APIResponse;
+use PKP\handler\APIHandler;
+use PKP\security\authorization\UserRolesRequiredPolicy;
+use PKP\security\Role;
+use PKP\security\authorization\PolicySet;
+use PKP\security\authorization\RoleBasedHandlerOperationPolicy;
+use Slim\Http\Request as SlimRequest;
+use Slim\Http\Response;
+
+class PKPJobHandler extends APIHandler
+{
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->_handlerPath = 'jobs';
+
+        $this->_pathPattern = '/admin/api/{version}/' . $this->_handlerPath;
```

Comment:
I think the URL pattern should be `/api/{version}/jobs`:
- Version number comes first (for consistency with other endpoints)
- `jobs` reflects the entity we're working with, rather than the person/role who is working with it

Reply:
I had dropped the ball on this earlier, but had to come back to it as part of #8706, which will forbid arbitrary values for the context path (like `admin`, which was introduced here). I've adjusted this to just use `index`, which is what we use elsewhere to indicate non-context-specific requests; see https://github.com/pkp/pkp-lib/pull/8717/commits/efaa7421768c3b3fb78c104de4bd10e509958955 (this link may change if I have to force-push).